pkgs/tools/security/otpgen/default.nix

```
@@ -0,0 +1,39 @@
+{ 
+  pkgs
+  , lib
+  , stdenv
+  , makeWrapper
+  , fetchFromGitHub
+  , cmake
+  , pkg-config
+  , cryptopp
+  , qtbase
+}:
+
+stdenv.mkDerivation {
+
+  name = "OTPGen";
```

```suggestion
  pname = "OTPGen";
  version = "0.9.3";
```
The package version should be in the metadata. This is the preferred convention right now